ajax/libs/jquery-animateNumber/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery-animateNumber",
+  "filename": "jquery.animateNumber.min.js",
+  "version": "0.0.14",
+  "dependencies": {
+    "jquery": ">= 1.7.0"
+  },
+  "description": "jQuery animate number plugin",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aishek/jquery-animateNumber.git"
+  },
+  "keywords": [
+    "jquery",
+    "animate",
+    "number",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "author": "Alexandr Borisov (aishek@gmail.com)",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/aishek/jquery-animateNumber.git",
+    "fileMap": [
+      {
+        "basePath": "libs",
+        "files": [
+          "jquery.animateNumber.js*"
```

Comment:
I think that `jquery.animateNumber*` is better, because there is another file we need (`jquery.animateNumber.min.js`).
Please modify `jquery.animateNumber.js*` to `jquery.animateNumber*`.